code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -108,18 +108,27 @@
 		if(100 to 200)
 			. += "<span class='warning'>[t_He] [t_is] twitching ever so slightly.</span>"
 
-	var/appears_dead = 0
+	var/appears_dead = FALSE
+	var/just_sleeping = FALSE
+
 	if(stat == DEAD || (HAS_TRAIT(src, TRAIT_FAKEDEATH)))
-		appears_dead = 1
-		if(suiciding)
-			. += "<span class='warning'>[t_He] appear[p_s()] to have committed suicide... there is no hope of recovery.</span>"
-		if(hellbound)
-			. += "<span class='warning'>[t_His] soul seems to have been ripped out of [t_his] body. Revival is impossible.</span>"
-		. += ""
-		if(getorgan(/obj/item/organ/brain) && !key && !get_ghost(FALSE, TRUE))
-			. += "<span class='deadsay'>[t_He] [t_is] limp and unresponsive; there are no signs of life and [t_his] soul has departed...</span>"
-		else
-			. += "<span class='deadsay'>[t_He] [t_is] limp and unresponsive; there are no signs of life...</span>"
+		appears_dead = TRUE
+
+		if(isliving(user))
+			var/mob/living/L = user
+			if(L.job == "Clown" && L.mind && L.mind.special_role)

```

Comment:
wait, are you supposed to use "L.mind.special_role" or "!L.mind.special_role" here? (I don't have much experience with special_role, hence why I'm asking)

Reply:
Special_role seems to be related to antag roles, i think he means !L.mind.special_role so syndicate/blood brother/survivalist/devil/etc. clowns know the truth

Reply:
Yep I forgot to push the right version last night oops!